pkgs/applications/audio/aaxtomp3/default.nix

```diff
@@ -45,7 +45,7 @@ resholve.mkDerivation rec {
       "bin/aaxtomp3"
       "bin/interactiveaaxtomp3"
     ];
-    interpreter = "${bash}/bin/bash";
+    interpreter = "${lib.getExe bash}";
```

Comment:
An LSP mage would be cool now!

Or a mere regex-replace...

Reply:
Just for context: I fully accept a "lets do that in another PR after this one" I just wanted to bring the idea up and talk about it for a second.

Reply:
I'm leaning towards new PR, since we already have some instances of `"${...}"` in nixpkgs that can be cleaned up at the same time, and we should be careful to only change those examples outside of an indented strinpg where we don't rely implicitly on the path -> string coercion.

Besides, I think this PR is quite large to review as is

Reply:
I did a `rg ' = "\$\{lib.getExe .*\}";' pkgs/ -l | xe sd ' = "\$\{lib.getExe (.*)\}";' ' = lib.getExe $1;'`